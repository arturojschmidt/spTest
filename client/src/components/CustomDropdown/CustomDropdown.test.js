
import React from 'react';
import { shallow } from 'enzyme';
import CustomDropdown from './CustomDropdown.js';

describe('CustomDropdown', () => {
 it('should be defined', () => {
   expect(CustomDropdown).toBeDefined();
 });
 it('should render correctly', () => {
   const tree = shallow(
     <CustomDropdown name='customdropdown test' />
   );
   expect(tree).toMatchSnapshot();
 });
});
