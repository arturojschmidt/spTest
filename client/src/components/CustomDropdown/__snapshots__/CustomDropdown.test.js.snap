// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomDropdown should render correctly 1`] = `ShallowWrapper {}`;
